packages/nvidia_nat_vanna/src/nat/plugins/vanna/vanna_utils.py

```
@@ -0,0 +1,845 @@
+    def assistant_message(self, message: str) -> dict:
+        """Create assistant message."""
+        return {"role": "assistant", "content": message}
+
+    def get_training_sql_prompt(
+        self,
+        ddl_list: list,
+        doc_list: list,
+    ) -> list:
+        """Generate prompt for synthetic question-SQL pairs."""
+        initial_prompt = (f"You are a {self.dialect} expert. "
+                          "Please generate diverse question–SQL pairs where each SQL "
```

Ruff (0.14.3)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)